src/commonmark-rules.js

```diff
@@ -66,12 +66,12 @@ rules.listItem = {
       .replace(/^\n+/, '') // remove leading newlines
       .replace(/\n+$/, '\n') // replace trailing newlines with just a single one
       .replace(/\n/gm, '\n    ') // indent
-    var prefix = options.bulletListMarker + '   '
+    var prefix = options.bulletListMarker + ' '
     var parent = node.parentNode
     if (parent.nodeName === 'OL') {
       var start = parent.getAttribute('start')
       var index = Array.prototype.indexOf.call(parent.children, node)
-      prefix = (start ? Number(start) + index : index + 1) + '.  '
+      prefix = (start ? Number(start) + index : index + 1) + '. '
```

Comment:
I think it would be easier to use 

```
prefix = `${start.padEnd(3)}.`
```

```
> "1".padEnd(3)
'1  '
> 12".padEnd(3)
'12 '
> "123".padEnd(3)
'123'
```

Reply:
Let's make minimal changes to the original codebase to make it easier to switch back to the original repo if the author returns or if we choose to use another fork.